libs/components/indicators/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -24,20 +25,35 @@ describe('Help inline component', () => {
   let fixture: ComponentFixture<HelpInlineTestComponent>;
   let cmp: HelpInlineTestComponent;
   let debugElement: DebugElement;
+  let logService: SkyLogService;
+  let deprecatedLogSpy: jasmine.Spy;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
       declarations: [HelpInlineTestComponent],
       imports: [BrowserModule, SkyHelpInlineModule],
     });
-
+    logService = TestBed.inject(SkyLogService);
+    deprecatedLogSpy = spyOn(logService, 'deprecated');
     fixture = TestBed.createComponent(HelpInlineTestComponent);
     cmp = fixture.componentInstance as HelpInlineTestComponent;
     debugElement = fixture.debugElement;
 
     fixture.detectChanges();
   });
 
+  it('should log that the component is deprecated', () => {
+    fixture.detectChanges();
+    expect(deprecatedLogSpy).toHaveBeenCalledWith(
+      'SkyHelpInlineComponent',
+      Object({
+        deprecationMajorVersion: 10,
+        replacementRecommendation:
+          'Use the help inline button component in the `@skyux/help-inline` library instead.',
```

Comment:
We're calling out the new `@skyux/help-inline` package, but all of its components/types are still marked as "internal".